internal/action/main_test.go

```diff
@@ -336,3 +337,88 @@ production:
 		assert.Len(t, v.Items, 1)
 	}
 }
+
+// findWorkspaceByName finds the first workspace matching the passed match string
+func findWorkspaceByName(name string, workspaceList *tfe.WorkspaceList) *tfe.Workspace {
+	for _, ws := range workspaceList.Items {
+		if strings.Contains(ws.Name, name) {
```

Comment:
My comment was about this and not just the name of the function. This implies that the program is not aware of the exact workspace name, but I would expect it would know exactly what the name is. Based on the way it's called it seems like this should be a simple equality check.

Reply:
oh yeah my bad, i updated the _usage_ to pass the full name but i didn't update the function body itself